cirq/google/xmon_gates.py

```diff
@@ -347,20 +344,6 @@ def __pow__(self, power):
         return ExpWGate(half_turns=self.half_turns * power,
                         axis_half_turns=self.axis_half_turns)
 
-    def try_cast_to(self, desired_type, ext):
-        if desired_type is ops.ReversibleEffect and self.has_inverse():
-            return self
-        return super().try_cast_to(desired_type, ext)
-
-    def has_inverse(self):
-        return not isinstance(self.half_turns, value.Symbol)
-
-    def inverse(self):
-        if not self.has_inverse():
-            raise ValueError("Don't have a known inverse.")
-        return ExpWGate(half_turns=-self.half_turns,
-                        axis_half_turns=self.axis_half_turns)
-
```

Comment:
Actually, ExpWGate does not have _pow_ that I can see.  Doesn't this represent a slight drop in functionality then?

Reply:
New doc looks fine.  Can you address this comment?  Seems like ExpWGate should have a pow implementation for -1?

Also, looks like build failed.

Reply:
ExpW's `__pow__` method is defined on line 344.

Reply:
Ok, I must be really tired, I went through the file twice and didn't see it either time.  LGTM then.